cmd/bpf2go/main.go

```diff
@@ -81,6 +81,7 @@ func run(stdout io.Writer, pkg, outputDir string, args []string) (err error) {
 	fs.StringVar(&b2g.makeBase, "makebase", "", "write make compatible depinfo files relative to `directory`")
 	fs.Var(&b2g.cTypes, "type", "`Name` of a type to generate a Go declaration for, may be repeated")
 	fs.BoolVar(&b2g.skipGlobalTypes, "no-global-types", false, "Skip generating types for map keys and values, etc.")
+	fs.StringVar(&b2g.filenameStem, "filename-stem", "", "alternative stem for names of generated files (defaults to ident)")
```

Comment:
Nit: Please call this `-output-stem` and call the new field `outputStem`, so it matches `outputDir`.

Reply:
There should be a check to make sure `b2g.outputStem` doesn't contain directory separators. Said another way `-output-stem ../foo` should be rejected. A test for this would also be nice.

Something like `filepath.Clean(outputStem) != outputStem` or similar.